docs/rules.md

```diff
@@ -1,4 +1,4 @@
 <!-- Generated with Stardoc: http://skydoc.bazel.build -->
 
-Public API re-exports
+
```

Comment:
that's surprising, why did that line go away?

Reply:
stardoc change I'm guessing

Reply:
https://github.com/bazel-contrib/rules-template/blob/main/mylang/defs.bzl#L1

the text is still there, so something feels broken here.

Reply:
ahh. if there are no exported symbols the generated file is empty now

Reply:
When I add
```
def example():
  """This is an example"""
  print("example export")
```
Then the .md is no longer empty